substrate/frame/support/test/tests/storage_transaction.rs

```diff
@@ -92,7 +92,7 @@ frame_support::construct_runtime!(
 impl frame_system::Config for Runtime {
 	type BaseCallFilter = frame_support::traits::Everything;
 	type Block = Block;
-	type BlockHashCount = ConstU32<10>;
+	type BlockHashCount = ConstU64<10>;
```

Comment:
You can just delete this line and let the `derive_impl` handle it.

Reply:
Yep, verified that `TestDefaultConfig` is set to use `ConstU64<10>` as default. Removed these, thanks for pointing it out